draft/webxdc-dev-reference.md

```diff
@@ -28,7 +28,7 @@ no need to add `webxdc.js` to your ZIP-file):
 window.webxdc.sendUpdate(update, descr);
 ```
 
-Webxdc apps are usually shared in a chat and run independently on each peer.
+A Webxdc is usually shared in a chat and run independently on each peer.
```

Comment:
Which you seem to believe here as well :smile:

Reply:
:)